fakeredis/_server.py

```diff
@@ -2164,14 +2164,17 @@ def zlexcount(self, key, min, max):
 
     def _zrange(self, key, start, stop, reverse, *args):
         zset = key.value
-        # TODO: does redis allow multiple WITHSCORES?
-        if len(args) > 1 or (args and not casematch(args[0], b'withscores')):
-            raise SimpleError(SYNTAX_ERROR_MSG)
+        withscores = False
+        for arg in args:
```

Comment:
Do you test this with multiple args somewhere?

Reply:
The Hypothesis-based fuzz tester does, which is what prompted this. I haven't bothered writing a targeted test since it's not something I expect any code in the wild to actually do.